### PR TITLE
Adapt Settings to the design and split the Display and Profile settings

### LINKED ISSUE #1907: Consistent Header + Menu Font Sizing
Small thing that I noticed while looking through the current stage is the inconsistency of text-sizing as pictured below:

**How it should be:**

We don't have to strictly follow the 16px or 20px, it's just a concern of consistent/relevant sizing. The settings tab headers should also be bolded depending on which tab they're on.

### PR DESCRIPTION
**Pull request by Tbaut**
closes #1907
The easiest is to play with it, and see if you find anything inconsistent.

---
Submission checklist: 

   

#### Layout
- [x] Change looks good in the desktop web ui
- [x] Change looks good in the mobile web ui

#### Theme
- [x] Components / elements inspected in light mode 
- [x] Components / elements inspected in dark mode

### CODE REVIEW

**Review by asnaith — APPROVED** [review]
Looks great and makes sense. It's nicer to use on mobile too 👍

File: packages/files-ui/src/Components/Elements/InvoiceLines.tsx [hunk 1]
```diff
@@ -56,11 +57,16 @@ const useStyles = makeStyles(
         color: palette.additional["volcano"][7]
       },
       price: {
-        fontWeight: "bold !important" as "bold"
+        fontWeight: "bold !important" as "bold",
```
Comment (tanmoyAtb) [inline packages/files-ui/src/Components/Elements/InvoiceLines.tsx:60]: Weird that the types don't recognize `!important` here

Comment (Tbaut) [reply packages/files-ui/src/Components/Elements/InvoiceLines.tsx:60]: Yup, the type checking is dumb here.

**Review by tanmoyAtb — APPROVED** [review]
Amazing amazing work, UX is much better. 

Just found this small margin offset, we might not keep this section anyway, still putting it here. 
The button on first and last name,

Screenshot 2022-02-11 at 8 23 19 PM

**Comment by Tbaut** [comment]
Thanks, whether or not we keep it, I guess we would only comment it out, so I'll def fix that now

**Review by FSM1 — APPROVED** [review]
Great work, this feels much smoother an experience🎉

**Comment by Tbaut** [comment]
Little heads up, I added a last minute small change to my settings PR, by removing the lock icon to the "Save change" button because IMHO a lock doesn't fit with saving a user name (this same icon is used on the security tab) and we don't use icons on buttons in general. lmk if you disagree. Also a note to myself, when disabled, this button has the same color as the background and looks totally out of place :) 

image

**Pull request merged**